packages/react-native-web/src/exports/render/index.js

```diff
@@ -9,46 +9,21 @@
 
 'use client';
 
-import {
-  hydrate as domLegacyHydrate,
-  render as domLegacyRender
-} from 'react-dom';
 import {
   createRoot as domCreateRoot,
   hydrateRoot as domHydrateRoot
 } from 'react-dom/client';
 
-import unmountComponentAtNode from '../unmountComponentAtNode';
 import { createSheet } from '../StyleSheet/dom';
 
 export function hydrate(element, root) {
   createSheet(root);
   return domHydrateRoot(root, element);
 }
 
-export function render(element, root) {
+export default function render(element, root) {
```

Comment:
You are importing a named `render` from renderApplication.js but here it's now a default export.

Reply:
ohhh, Thanks I will fix that.